connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorPluginsResource.java

```diff
@@ -133,6 +135,7 @@ public ConfigInfos validateConfigs(
 
     @GET
     @Path("/")
+    @Operation(summary = "List all connector plugins installed")
```

Comment:
We should explain the behaivour of the `connectorsOnly` parameter too

Reply:
Done